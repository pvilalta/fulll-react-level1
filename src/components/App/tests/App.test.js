import React from 'react';
import { render } from '@testing-library/react';
import Component from '..';

describe('components | App | index', () => {
  test('should render component', () => {
    const { container } = render(<Component />);
    expect(container).toMatchSnapshot();
  });
});
