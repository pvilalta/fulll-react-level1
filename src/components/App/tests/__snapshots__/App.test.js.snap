// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | App | index should render component 1`] = `
<div>
  <div
    class="App"
  >
    <div
      class="Playground"
    >
      <div
        class="boxList"
      >
        <div
          class="titleBox"
        >
          <span>
            Tick the checkboxes
          </span>
        </div>
        <div
          class="welcomeBox"
        />
        <div
          class="selectAllBox"
        >
          <input
            data-testid="selectAll"
            id="selectAll"
            name="selectAll"
            type="checkbox"
          />
          <label
            for="selectall"
          />
        </div>
        <div
          class="listItemsBox"
        >
          <div
            class="item"
          >
            <input
              data-testid="item-0"
              id="item-0"
              name="item-0"
              type="checkbox"
            />
            <label
              for="item-0"
            />
          </div>
          <div
            class="item"
          >
            <input
              data-testid="item-1"
              id="item-1"
              name="item-1"
              type="checkbox"
            />
            <label
              for="item-1"
            />
          </div>
          <div
            class="item"
          >
            <input
              data-testid="item-2"
              id="item-2"
              name="item-2"
              type="checkbox"
            />
            <label
              for="item-2"
            />
          </div>
          <div
            class="item"
          >
            <input
              data-testid="item-3"
              id="item-3"
              name="item-3"
              type="checkbox"
            />
            <label
              for="item-3"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
